Dockerfile.prod

```diff
@@ -1,19 +1,21 @@
-# NodeJS Version 16
-FROM node:16.18-buster-slim
-
-# Copy Dir
-COPY . ./app
+# NodeJS Version 18
+FROM node:18-alpine
 
 # Work to Dir
 WORKDIR /app
 
+COPY ./package.json ./package-lock.json /app/
+
 # Install Node Package
 RUN npm install --legacy-peer-deps
 
 # Set Env
-ENV NODE_ENV production
+ENV NODE_ENV development
```

Comment:
why development when this is a prod dockerfile

Reply:
updated